routes/index.js

```diff
@@ -1,11 +1 @@
-import express from 'express';
-
-const router = express.Router();
-
-router.get('/', (req, res) => {
-  res.status(200).json({
-    message: 'Hello Books Deferral'
-  });
-});
-
-export default router;
```

Comment:
Hey, @abdulfatai360 Great job setting up the automated testing for the project! However, is there any particular reason why all the routing logic in this file was deleted?

Reply:
Thanks for this feedback @tolulope-od . I deleted the current routing logic because I feel it was implemented wrongly (_or perhaps let me say I didn't understand it_). I found this out when I wanted to write test for the `root route`. You would notice that on the application's index file, the router instance exported from the router's index file was mounted on `api/v1/` URL path - so it makes the root endpoint URL to be `http://localhost:4000/api/v1`.

In my opinion and from what I have seen over the web, this could not be considered as the root of our app. So I felt I should change the logic to make the URL of the root be `http://localhost:4000`. However, I want you to note that the `routes/` directory wasn't deleted. This is because the directory will still be useful when we handle subsequent endpoints.

Also, I feel the implementation of the routes to be a sort of "progressive enhancement" (_i.e. starting from the app `index` file and then moving to the `routes/` directory as needs arise_). With this method, I think every team member will understand a bit of the routes/endpoints logic implementation (_I know my team mates are gurus though_)